libs/pubsub/pubsub.go

```diff
@@ -101,7 +101,7 @@ type Server struct {
 	cmdsCap int
 
 	mtx           sync.RWMutex
-	subscriptions map[string]map[string]Query // subscriber -> query (string) -> Query
+	subscriptions map[string]map[string]struct{} // subscriber -> query (string) -> empty struct
```

Comment:
Do I understand this correctly: The empty struct is only necessary here because we want to be able to quickly add / delete a queries in subscribe/unsubscribe (and maps are cool for this)?